db-checkbook/build_scripts/01_dataloading.py

```diff
@@ -1,29 +1,62 @@
-from dcpy.connectors import s3
 import pandas as pd
+import geopandas as gpd
+import boto3
+import os
+from pathlib import Path
+from dotenv import load_dotenv
 
 BASE_BUCKET = 'edm-recipes'
 BASE_URL = "https://edm-recipes.nyc3.cdn.digitaloceanspaces.com"
 
+_curr_file_path = Path(__file__).resolve()
+LIB_DIR = _curr_file_path.parent.parent / '.library'
+load_dotenv(_curr_file_path.parent.parent.parent / '.env') 
 
 
-def read_s3_edm_recipes_cpdb(version, type_geom, save_file_path):
+
+def download_s3_edm_recipes_cpdb():
```

Comment:
@damonmcc @alexrichey - having some difficulties with the cpdb zipped shapefiles (aren't happy being opened by gdal as part of data library or by geopandas here, so ) so we've gone a slightly unconventional route. @DeaBardhoshi had manually put the shapefiles in DO/edm-recipes as directories. This folder then recursively downloads the `dcp_cpdb` folder in DO/edm-recipes into the local filesystem, and geopandas then calls `read_file` on the folders themselves (containing all the relevant files for a shapefile) rather than a single file

Reply:
Yeah that's in 02_build